Simplified/Network/NYPLNetworkExecutor.swift

```diff
@@ -108,80 +76,102 @@ extension NYPLNetworkExecutor {
     return urlRequest
   }
 
-  @objc func clearCache() {
-    urlSession.configuration.urlCache?.removeAllCachedResponses()
+  /// Performs a GET request using the specified URL
+  /// - Parameters:
+  ///   - reqURL: URL of the resource to GET.
+  ///   - completion: Always called when the resource is either fetched from
+  /// the network or from the cache.
+  func GET(_ reqURL: URL,
+           completion: @escaping (_ result: NYPLResult<Data>) -> Void) {
+    let req = request(for: reqURL)
+    executeRequest(req, completion: completion)
   }
 }
 
-// Objective-C compatibility
-extension NYPLNetworkExecutor: NYPLNetworkExecuting {
-  @objc class func bearerAuthorized(request: URLRequest) -> URLRequest {
```

Comment:
this logic is already captured in the `request(for:)` method, so it's another duplication that I just removed. Now all requests in the executor are prepared with `request(for:)` and executed with `executeRequest(:completion:)`.